### ISSUE TITLE
goseg: Login screen gets stuck

### ISSUE THREAD

**Issue by thelifeandtimes** [Native-Planet/GroundSeg#437]
I get the login screen, but when I click submit, nothing happens (at least in the ui). Console pukes out a bunch of this:
![image](https://github.com/Native-Planet/GroundSeg/assets/82244264/e4288555-2545-4a85-b8fc-3a3362eea7cd)

If i then `ctrl + R` to refresh, it puts me into the authenticated ui. 

Expected behavior is for the submit button to put me into the authenticated UI.

Browser: Arc Browser on MacOS Sonoma.

**Comment by yapishu** [comment]
i merged a refactor of the auth and websocket management at the very moment you submitted this, would you mind restarting gs and then seeing if `sha256sum /opt/nativeplanet/groundseg/groundseg` produces `8068d95a3df3[...]`?

**Comment by yapishu** [comment]
also very weird that you're getting those printed in the console but not seeing anything in the ui since those messages are what prompt it to load the authenticated interface

**Comment by thelifeandtimes** [comment]
I just rebooted, checksum is `49f5578...` will try again in a little bit.

 It also seems like it hits some bug where it kicks me back out to the login page. a few other errors showing up in the console:
![image](https://github.com/Native-Planet/GroundSeg/assets/82244264/a42e49ff-08cc-424c-b2ac-035b1cf32c8c)

**Comment by nallux-dozryl** [comment]
that lockdown error is a red herring. Noticed that a bunch of "setup" blobs getting sent. Probably because stage is empty. Likely we missed something when deprecating "firstBoot" in favor of "setup" in goseg. Will add this to bug list

**Comment by nallux-dozryl** [comment]
To fix your current situation: 

1. In `/opt/nativeplanet/groundseg/settings/system.json`, replace whatever is in "setup" with "complete". `"setup":"complete"`.
2. Restart GroundSeg

**Comment by thelifeandtimes** [comment]
I'm still at `49f5578...`. Although realizing this bit of context is in a different issue, I'm getting updates from edge / staging:

```
    },
    "setup": "complete",
    "swapFile": "/opt/nativeplanet/groundseg/swapfile",
    "swapVal": 16,
    "updateBranch": "edge",
    "updateInterval": 3600,
    "updateMode": "auto",
    "updateUrl": "https://staging.version.groundseg.app",
    "wgOn": true,
    "wgRegistered": true
}
```

**Comment by thelifeandtimes** [comment]
Groundseg seems caught up now. Aside from currently replaying 29M events, looks like this issue is rectified.

**Comment by nallux-dozryl** [comment]
Glad to hear it's rectified. Permanent fix implemented as well #444
